src/migration-scripts/interfaces/16-to-17

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+# Command line migration of port mirroring
+# https://phabricator.vyos.net/T3089
+
+import sys
+from vyos.configtree import ConfigTree
+
+if __name__ == '__main__':
+    if (len(sys.argv) < 1):
+        print("Must specify file name!")
+        sys.exit(1)
+
+    file_name = sys.argv[1]
+
+    with open(file_name, 'r') as f:
+        config_file = f.read()
+
+    config = ConfigTree(config_file)
+    base = ['interfaces']
```

Comment:
No need for the iteration over all interface types, `mirror` was only available to `ethernet` interfaces before so you can keep this loop tight.